ticketconfig.py

```diff
@@ -52,9 +52,13 @@ def _setup_providers(self):
             ))
         p.append( h.GitlabTitleProvider( 'gitlab.torproject.org',
             'https://gitlab.torproject.org/',
-            prefix='gitlabtpo:',
+            prefix='tor:',
             default_re=r'(?<!\w)(?:gitlabtpo:)(?P<path>\w+/[\w/]*\w)#(?P<number>[0-9]+)(?:(?=\W)|$)',
             ))
+        p.append( h.GitlabTitleProvider( 'gitlab.torproject.org-legacy',
+            'https://gitlab.torproject.org/legacy/trac/-/issues/',
+            prefix='gitlabtpolegacy:',
+            ))
```

Comment:
The gitlabtitleprovider that you helped write takes tuples in its matches for project and then issue number.  I suspect we might just want a html provider.